docs/sources/features/datasources/jaeger.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Using Jaeger in Grafana"
+description = "Guide for using Jaeger in Grafana"
+keywords = ["grafana", "jaeger", "guide", "tracing"]
+type = "docs"
+aliases = ["/docs/grafana/latest/datasources/jeager"]
+[menu.docs]
+name = "Jaeger"
+parent = "datasources"
+weight = 2
++++
+
+# Using Jager in Grafana
+
+Grafana ships with built-in support for Jaeger, open source, end-to-end distributed tracing.
+Just add it as a data source and you are ready to query your traces in [Explore]({{< relref "../explore" >}}).
+
+## Adding the data source
```

Comment:
Change the top portion of this topic to match what is in Prometheus: https://grafana.com/docs/grafana/latest/features/datasources/prometheus/

All you need is a link to **Add a data source**, instructions for accessing Jaeger settings, and the table with the settings.

Btw, please add ALL the Jaeger settings and their descriptions to the table, not just these three.